elements/pfe-tabs/src/pfe-tab.scss

```diff
@@ -98,42 +98,48 @@ $LOCAL-VARIABLES: (
 
 @media screen and (min-width: #{pfe-breakpoint(md)}) {
   :host([vertical]) {
-    --pfe-tabs__tab--BorderTop: #{pfe-var(ui--border-width)} #{pfe-var(ui--border-style)} transparent;
-    --pfe-tabs__tab--BorderRight: 0;
-    --pfe-tabs__tab--BorderBottom: #{pfe-var(ui--border-width)} #{pfe-var(ui--border-style)} transparent;
-    --pfe-tabs__tab--BorderLeft: #{pfe-var(ui--border-width)} #{pfe-var(ui--border-style)} transparent;
-    --pfe-tabs__tab--Margin: 0 -1px 0 0;
-    --pfe-tabs__tab--PadingLeft: calc(#{pfe-var(container-padding)} * .75);
-    --pfe-tabs__tab--TextAlign: left;
-
-    @include browser-query(ie11) {
-      border: 1px solid transparent;
-      border-right: 0;
-      margin: 0 -1px 0 0;
-      padding-left: 12px;
-      text-align: left;
-    }
+    --pfe-tabs--BorderTop: #{pfe-var(ui--border-width)} #{pfe-var(ui--border-style)} transparent;
+    --pfe-tabs--BorderRight: 0;
+    --pfe-tabs--BorderBottom: 0;
+    --pfe-tabs--BorderLeft: #{pfe-var(ui--border-width)} #{pfe-var(ui--border-style)} transparent;
+    --pfe-tabs--Margin: 0 -1px 0 0;
+    --pfe-tabs--PadingLeft: calc(#{pfe-var(container-padding)} * .75);
+
+      @include browser-query(ie11) {
+        border: 1px solid transparent;
+        border-bottom: 0;
+        border-right: 0;
+        margin: 0 -1px 0 0;
+        padding-left: 12px;
+      }
   }
 
   :host([vertical][aria-selected="true"]) {
     --pfe-tabs__tab--BorderTopColor: transparent;
-    --pfe-tabs__tab--BorderRight: #{pfe-var(surface--border-width--heavy)} #{pfe-var(surface--border-style)} #{pfe-local(highlight)};
+    --pfe-tabs__tab--BorderRight:    #{pfe-local(BorderWidth--hover)} #{pfe-var(surface--border-style)} #{pfe-local(highlight)};
+    --pfe-tabs__tab--BorderBottom:   0;
 
     @include browser-query(ie11) {
-      // color: #{$pfe-color--black};
+      color:            #{$pfe-color--black};
       border-top-color: transparent;
-      border-right: #{pfe-var(surface--border-width--heavy)} #{pfe-var(surface--border-style)} #{pfe-var(ui-accent)};
+      border-right:     #{pfe-fetch(surface--border-width--heavy)} #{pfe-var(surface--border-style)} #{pfe-var(ui-accent)};
+      border-bottom:    0;
     }
   }
 }
 
 :host([pfe-variant="earth"]) {
   @include browser-query(ie11) {
-    background-color: #{pfe-var(surface--lighter)};
-    color: #{pfe-var(text)};
+    background-color: #{pfe-fetch(surface--lighter)};
+    color:            #{pfe-fetch(ui-disabled--text)}; // @todo replace with pfe(text-muted) in Kendall's color story
```

Comment:
you can go ahead and add the text-muted vars to the `_colors.scss` file in this branch:

```
  text:                        $pfe-color--text,
  text--muted:                 $pfe-color--gray-600,
  link:                        $pfe-color--link,
  link--visited:               $pfe-color--link--visited,
  link--hover:                 $pfe-color--link--hover,
  link--focus:                 $pfe-color--link--focus,

  text--on-dark:               $pfe-color--text--on-dark,
  text--muted--on-dark:        $pfe-color--gray-50,
  link--on-dark:               $pfe-color--link--on-dark,
  link--visited--on-dark:      $pfe-color--link--visited--on-dark,
  link--hover--on-dark:        $pfe-color--link--hover--on-dark,
  link--focus--on-dark:        $pfe-color--link--focus--on-dark,

  text--on-saturated:          $pfe-color--text--on-saturated,
  text--muted--on-saturated:   $pfe-color--gray-50,
  link--on-saturated:          $pfe-color--link--on-saturated,
  link--visited--on-saturated: $pfe-color--link--visited--on-saturated,
  link--hover--on-saturated:   $pfe-color--link--hover--on-saturated,
  link--focus--on-saturated:   $pfe-color--link--focus--on-saturated,
```